collect_app/src/main/java/org/odk/collect/android/activities/GeoTraceActivity.java

```diff
@@ -473,8 +484,18 @@ public void setGeoTraceMode(View view) {
     }
 
     public void setGeoTraceScheduler(long delay, TimeUnit units) {
```

Comment:
f I'm understanding correctly, if there's a time remaining, we know that we have rebuilt the activity and so we schedule a first vertex addition to happen after that time remaining has passed. Then we schedule another task which after the time remaining has elapsed schedules a fixed rate task for getting points. Does this sound right? It's a lot of schedulers so rather confusing!

Did you consider any other approaches? For example:

```
               schedulerHandler = scheduler.scheduleAtFixedRate(
                        () -> runOnUiThread(() -> addVertex()), timeRemaining, delay, units);
```

would mean that the first event would happen after `timeRemaining` as desired after the activity is rebuilt. It results in a slight change in behavior because when the collection is first scheduled the first point would be immediately captured rather than after an initial delay. I think this is actually what users expect and it's part of [the idealized spec](https://docs.google.com/document/d/1r3wJrOVDeFQkk4kWMoW29LtLI1vbBChqJ65w1Jyg4YU/edit) put together a little while ago. What do you think of going with that?